exam/src/main/java/in/testpress/exam/ui/TestInstructions.kt

```diff
@@ -0,0 +1,44 @@
+package `in`.testpress.exam.ui
+
+import `in`.testpress.exam.R
+import `in`.testpress.ui.BaseFragment
+import android.os.Bundle
+import android.text.Html
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.TextView
+
+
+class TestInstructions(val startExam: () -> Unit) : BaseFragment() {
+
+    companion object {
+        val INSTRUCTIONFLAG = "exam_instructions"
+    }
+
+    private lateinit var instructionsView: TextView;
+    private lateinit var confirmButton: Button
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+    }
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
+                              savedInstanceState: Bundle?): View? {
+        val view = inflater.inflate(R.layout.fragment_test_instructions, container, false)
+        initViews(view)
+
+        return view
+    }
+
+    private fun initViews(view: View){
```

Comment:
Initviews will make sense if you have many views and doing different operations. In this case you are just displaying instructions.
So you can have 2 methods instead of initViews
`displayInstructions`
`setListeners`